components/playlist/browser/playlist_service.cc

```diff
@@ -672,6 +666,33 @@ void PlaylistService::DeleteAllPlaylistItems() {
   CleanUpOrphanedPlaylistItemDirs();
 }
 
+void PlaylistService::RemoveLocalDataForItem(
+    const mojom::PlaylistItemPtr& item) {
+  DCHECK(item);
+  if (!item->cached)
+    return;
+
+  if (!item->media_path.SchemeIsFile())
+    return;
+
+  item->cached = false;
+  std::string path_string = item->media_path.spec();
+  base::FilePath media_path(std::string(
+      path_string.begin() + 7 /*exclude file://*/, path_string.end()));
```

Comment:
how about using `base::RemoveChars(src, "file://", out)`?

Reply:
Oh, didn't know that. Thanks.